src/scripts/main.js

```diff
@@ -1,3 +1,141 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const tableBody = document.createElement('tbody');
+table.innerHTML = '';
+table.append(tableBody);
+
+const tbody = document.querySelector('tbody');
+
+const apppendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+const tableSize = {
+  width: 4,
+  height: 4,
+};
+
+const buttonsState = {
```

Comment:
Use boolean values for button states: true for 'enabled' and false for 'disabled'.